src/services/ledger/tx-metadata.ts

```diff
@@ -94,7 +94,9 @@ export const OnChainSendLedgerMetadata = <
 
     fee: toSats(satsFee),
     feeUsd: convertCentsToUsdAsDollars(feeDisplayCurrency),
-    usd: convertCentsToUsdAsDollars(amountDisplayCurrency),
+    usd: convertCentsToUsdAsDollars(
+      (amountDisplayCurrency + feeDisplayCurrency) as DisplayCurrencyBaseAmount,
```

Comment:
are we adding fee in centsAmount?

Reply:
Nope, satsAmount and centsAmount are without fees